addons/custom-menu-bar/addon.json

```
@@ -0,0 +1,127 @@
+{
+  "name": "Customizable menu bar",
+  "description": "Allows you to hide specific editor menu bar items or remove their labels or icons.",
+  "credits": [
+    {
+      "name": "philipp2007",
+      "link": "https://ggigabyte.repl.co/"
+    },
+    {
+      "name": "CST1229",
+      "link": "https://github.com/CST1229/",
+      "note": "compact user dropdown",
+      "id": "dropdown"
+    },
+    {
+      "name": "lisa_wolfgang",
+      "link": "https://scratch.mit.edu/users/lisa_wolfgang/",
+      "note": "hide tutorials button",
+      "id": "tutorials-button"
+    }
+  ],
+  "info": [
+    {
+      "id": "tutorials-button-update",
+      "text": "The \"Hide Tutorials button\" addon has been moved here as a setting."
+    }
```

Just in case of confusion:
```suggestion
    },
    {
      "id": "small-screens",
      "text": "The text labels may be automatically replaced by icons on smaller windows."
    }
```